ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1368,6 +1368,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: e2e-azure-ipi-sdn-migration-ovn-p2
+  cron: 0 10 */28 * *
```

Comment:
p2 runs every 7 days (*/7), and p3 runs every 28 days (*/28). Perhaps make them match each other ?

Reply:
@liangxia Fixed it.